Update to Bitcoin Design Crits from Design Review (#341)
BitcoinDesign/Meta: Bitcoin-Design-Crits.md

# Bitcoin Design Crits

## What are Bitcoin Design Crits?

Bitcoin Design Crits are opportunities for bitcoin products to get design critiques (crits) on a particular page or section in their product. Actionable suggestions for improving usability, user interface, interaction, and visual aesthetics are given by members in the Bitcoin Design Community.

## How does it work?

> 1. A bitcoin product submits a page or section they want reviewed on Bitcoin Design Crits in the #design-review in our Slack.

Reviewer: ```suggestion
1. A bitcoin product submits a page or section they want reviewed on Bitcoin Design Crits in the [#design-review](https://bitcoindesign.slack.com/archives/C019MTNFKL7) channel in our Slack.
```

We will only focus on a particular page or section of your product so we can give the highest amount of value. Reviewing a whole product is an extensive and involved process that is better suited to be dealt with by a dedicated Product Designer. If you need a Product Designer for your bitcoin product, reach out in the opportunities channel in our Slack.

2. A product is selected for the next upcoming Bitcoin Design Crit call by members of the community.

> 3. Product designers interested in reviewing jump into a collaborative Figma file where they crit the product page or section submitted **PRIOR** to the Bitcoin Desisgn Crit call. This will all be public and open for anyone to view in classic open-source fashion.

Reviewer: ```suggestion
3. Product designers interested in reviewing jump into a collaborative Figma file **PRIOR** to call, where they crit the product page or section submitted. This will all be public and open for anyone to view, in classic open-source fashion.
```
Edit for clarity. Was not clear whether the submission or the crit'ing should happen before the call.

> 4. Product designers who crit the product present their feedback to the product team in the scheduled Bitcoin Design Crit call which happen Thursdays at 2:00PM UTC.

Reviewer: ```suggestion
4. Product designers who crit the product present their feedback to the product team in the scheduled Bitcoin Design Crit calls, which happen Thursdays at 2:00PM UTC.
```

Follow up help and suggestions on the feedback given is always available.

## Who can take part?

Anyone who is working on a bitcoin product. Whether its a new page you want reviewed before release, or an old page that needs polishing, Bitcoin Design Crits can help. Open-source products get priority over closed-source products.

Calls are recorded and open for anyone to join in and view the crit take place. We greatly appreciate the pseudonymous and private nature of bitcoin and are more than happy to accommodate those who would prefer not to reveal their identity.

## What information do I need to provide for a crit?

> - The section or page you want reviewed. We can help you identify an area that needs work if you can't

Reviewer: ```suggestion
- The section or page you want reviewed. We can help you identify an area that needs work if you're not sure
```

- Links to any design files you use in your product if you have them: Figma file, design system, icon set, band assets

> - If your app has a tesnet or beta version that needs special access to join that would make the process a lot easier! 

Reviewer: ```suggestion
- Access information, if your app has a testnet or beta version that requires authentication to use 
```

## What we won't do

> - Make up new features or suggest developmental changes (unless necessary, relevant, or just as an optional thing to do look into)

Reviewer: ```suggestion
- Make up new features or suggest developmental changes (unless necessary, relevant, or just as an optional thing to look into)
```
